src/Resources/Resources/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+
+## Version 1.1.2
 * Fix for issue https://github.com/Azure/azure-powershell/issues/8166
```

Comment:
This is ok for right now, but we should update this description to include a short summary of the issue fixed along with the link.